src/gui/guiFormSpecMenu.cpp

```diff
@@ -2648,16 +2802,27 @@ void GUIFormSpecMenu::regenerateGui(v2u32 screensize)
 
 	pos_offset = v2f32();
 
+	// used for formspec versions < 2
+	core::list<IGUIElement *>::Iterator legacy_sort_start = Children.getLast();
+
 	if (enable_prepends) {
 		// Backup the coordinates so that prepends can use the coordinates of choice.
 		bool rc_backup = mydata.real_coordinates;
-		bool version_backup = m_formspec_version;
+		u16 version_backup = m_formspec_version;
```

Comment:
this should be fixed on master before this PR is merged

Reply:
Done.